app/models/user.php

```diff
@@ -82,5 +81,148 @@ public function login(LoginInfo $login_info)
             throw new RecordNotFoundException('No Record Found');
         }
         Session::set('username', $this->username);
+        Session::set('id', $row['id']);
+    }
+
+    public static function getAll()
+    {
+        $user = array();
+        $db = DB::conn();
+        $rows = $db->rows("SELECT * FROM user WHERE id = ?", array(Session::get('id')));
+
+        foreach ($rows as $row) {
+            $user[] = new self($row);
+        }
+        return $user;
+    }
+
+    public static function getComments()
```

Comment:
getComments() should be in the comments model :bow: 
Also I think it would be better if you would just use 1 function, and then just pass an argument for the id (so you can use for either the current user and the other users po :dart: